docs/how-to-test.md

```diff
@@ -63,8 +63,8 @@ To deploy the Gravitino server locally to run the integration tests, follow thes
    in the `distribution` directory.
 3. Use the `./gradlew test [--rerun-tasks] -PskipTests -PtestMode=deploy` command to run the
    integration tests in the `distribution` directory.
-4. Use the `bash integration-test/trino-test-tools/trino_test.sh` command to run all the
-   Trino test sets in the `integration-test/src/test/resources/trino-ci-testset/testsets` directory.
+4. Use the `bash trino-connector/integration-test/trino-test-tools/trino_test.sh` command to run all the
+   Trino test sets in the `trino-connector/integration-test/src/test/resources/trino-ci-testset/testsets` directory.
 
```

Comment:
The command is still works for the changes

Reply:
yes